.dockerignore

```diff
@@ -0,0 +1,7 @@
+README.Rmd
+README.md
+docs/
+input/
+renv/library/
+renv/sandbox/
+renv/staging/
```

Comment:
Ignoring this when using `COPY` in Docker since it's manually replaced with the installed libraries.